src/network/protocol/messages/lib.zig

```diff
@@ -60,13 +62,49 @@ pub const MessageTypes = enum {
     sendcmpct,
     feefilter,
     filterclear,
+    getdata,
     block,
     filteradd,
     notfound,
     sendheaders,
     filterload,
 };
 
+pub const InventoryItem = struct {
+    type: u32,
+    hash: [32]u8,
+
+    pub fn serialize(self: *const InventoryItem, w: anytype) !void {
```

Comment:
can you name it encodeToWriter, in order to be coherent across the different structs